performance/odm/playlist.xml

```diff
@@ -28,6 +30,5 @@
 		<groups>
 			<group>perf</group>
 		</groups>
-		<disabled>https://github.com/AdoptOpenJDK/openjdk-tests/issues/457</disabled>
```

Comment:
ODM should not run in AdoptOpenJDK. If we remove `<disabled>`, please leave a comment if test material does not exist.

Reply:
I thought we can run ODM in AdoptOpenJDK as long as the perf machines have the benchmark material. From this comment (https://github.com/AdoptOpenJDK/openjdk-tests/issues/457#issuecomment-401170759), seems like 2 perf machines already have the material. Please let me know if that's not the case and I can disable this benchmark.

Reply:
No, this is not the case. We should run ODM internally only, at least for now.
I did not get chance to remove the perf material on these two machines. If you can help to delete them, that would be great. Thanks.
We should still enable the test, so we can run internally.

Reply:
ODM benchmark zip files have been removed from both machines.

Reply:
@llxia Sounds good! I'll disable it! Btw in order to disable it for AdoptOpenJDK and enable it internally, I guess we'll need to have an internal repo with `odm/build.xml` instead `odm/exclude_build.xml` and no `<disabled>` element in `odm/playlist.xml` right? Not sure if there's a better way of doing this.

Reply:
no, we do not need to have an internal repo.
In all perf builds in Adopt, we just need to set `BUILD_LIST=performance/idle_micro` for now.
Your current change for enabling test is fine.

Reply:
Perf jobs are updated to use `BUILD_LIST=performance/idle_micro`
Tested manually:
https://ci.adoptopenjdk.net/view/Test_perf/job/openjdk11_hs_perftest_x86-64_linux/85/